src/mongo/tree/MongoDatabaseTreeItem.ts

```diff
@@ -146,6 +147,21 @@ export class MongoDatabaseTreeItem implements IAzureParentTreeItem {
 	}
 }
 
+export function validateCollectionName(collectionName: string): string | undefined | null {
+	// https://docs.mongodb.com/manual/reference/limits/#Restriction-on-Collection-Names
+	if (collectionName.length === 0) {
```

Comment:
if (!collectionName) ...

I've seen cases where we can get called with an undefined value and that would throw.